### PR TITLE
create team

### PR DESCRIPTION
**Pull request by NguyenHai2310**

### CODE REVIEW

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    @team = Team.find params[:id]
+  end
+
+  def new
+    @team = Team.new
+  end
+
+  def create
+    @team = Team.new team_params
+    #debugger
```
Comment (phonghenry14) [inline app/controllers/admin/teams_controller.rb]: xóa cmt

File: app/controllers/admin/teams_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  end
+
+  def edit
+    @team = Team.find params[:id]
+  end
+  
+  def update
+    @team = Team.find params[:id]
+    if @team.update_attributes team_params
+      flash[:notice]= t "team.update.success"
```
Comment (phonghenry14) [inline app/controllers/admin/teams_controller.rb]: dùng success vs danger

File: app/views/admin/teams/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class='row' >
+  <div class='col-md-6 col-md-offset-3' >
+    <%= form_for [:admin, @team] do |f| %>
+      <%= render 'shared/error_messages', object: f.object %>
```
Comment (phonghenry14) [inline app/views/admin/teams/_form.html.erb]: " "

File: app/views/admin/teams/_team.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= link_to team.name, admin_team_path(team) %></td>
+  <td>|<%= link_to " ", edit_admin_team_path(team), class: "glyphicon glyphicon-edit" %></td>
+  <td>|<%= link_to " ", admin_team_path(team), 
+  method: :delete, data: {confirm: "You sure?"}, class: "glyphicon glyphicon-trash" %></td>
```
Comment (phonghenry14) [inline app/views/admin/teams/_team.html.erb]: i18n

File: app/views/admin/teams/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<% provide :title, t("team.index.title") %>
+<h1><%= t "team.index.title" %></h1>
+<p><%= link_to "Create New Team", new_admin_team_path, class: "btn btn-success" %></p>
```
Comment (phonghenry14) [inline app/views/admin/teams/index.html.erb]: i18n

File: app/controllers/admin/teams_controller.rb [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+  def destroy
+    @team = Team.find(params[:id]).delete
+    flash[:success] = t "team.destroy"
+    redirect_to admin_teams_path
+  end
+
+  private
+  def team_params
+  params.require(:team).permit :name, :description
```
Comment (HoangVanTrinh92) [inline app/controllers/admin/teams_controller.rb]: ident

**Pull request merged**